### ISSUE TITLE
Function isArrayOfStrings translations AR, ES, FR, IT, HR

### ISSUE THREAD

**Issue by ilia-brykin** [ilia-brykin/aloha#122]
We need translations of Documentation for isArrayOfStrings: AR, ES, FR, IT, HR

URL: https://ilia-brykin.github.io/aloha/#/utils-is-array-of-strings

**English words in a non-English translation should be highlighted for screen reader**
`<span lang="en"></span>` or `<strong lang="en"></strong>`

**Comment by ilia-brykin** [comment]
EN, DE, RU done.

**Comment by shafedh** [comment]
ready to test

**Comment by ilia-brykin** [comment]
AR, ES, FR, IT done.

**Comment by zdilarr** [comment]
HR done

**Comment by babeyeva** [comment]
Ок
